Resources/Computing/GlobusComputingElement.py

```diff
@@ -67,35 +69,9 @@ def submitJob( self, executableFile, proxy, numberOfJobs = 1 ):
     for _i in xrange(numberOfJobs):
       diracStamp = makeGuid()[:8]
       queueName = '%s/%s' % ( self.ceName, self.queue )
-      #cmd = ['globus-job-submit', queueName, "-s", executableFile ]
-      cmd = ['globus-job-submit', queueName]
-      ## Add extra options
-      queueCount = self.ceParameters['QueueCount']
-      queueHostCount = self.ceParameters['QueueHostCount']
-      queuePar = self.ceParameters['QueueParameter']
-      queueMaxTime = self.ceParameters['maxCPUTime']
-      #queueOut = self.ceParameters['outLog']
-      #queueErr = self.ceParameters['errorLog']
-
-      #self.log.verbose(queueCount)
-      #self.log.verbose(queueHostCount)
-      #self.log.verbose(queuePar)
-
-      if queueCount:
-        cmd += ['-count',queueCount]
-      if queueHostCount:
-        cmd += ['-host-count',queueHostCount] 
-      if queuePar:
-        cmd += ['-q',queuePar]
-      if queueMaxTime:
-        cmd += ['-maxtime', queueMaxTime]
-      #if queueOut:
-      #  cmd += ['-stdout', queueOut]
-      #if queueErr:
-      #  cmd += ['-stderr', queueErr]
-      cmd += ["-s", executableFile ]
+      
+      cmd = ['globus-job-submit', queueName, " ", self.extraSubmitString, "-s", executableFile ]
```

Comment:
Why dropping all the previous options?

Reply:
Easier to add everything in one inclusive cfg.

Reply:
Why the empty `" ",` entry in the cmd list?